.openshift-ci/crawler/copy-init.sh

```diff
@@ -0,0 +1,71 @@
+#!/usr/bin/env bash
```

Comment:
[ultra-nit] `copy-init.sh` is a little misleading here, since it is actually copying and running.